test/e2e/specs/web_app.js

```diff
@@ -0,0 +1,38 @@
+
+module.exports = {
+  'Demo WebbApp test' : function (browser) {
+    browser
+      .url('https://repartyed.firebaseapp.com')
+      .waitForElementVisible('body', 5000)
+
+      .waitForElementVisible('#app > div.application--wrap > div > main > div.container.fluid.fill-height > div > div > div > div > div > div.card__actions > button > div', 5000)
+      .click('#app > div.application--wrap > div > main > div.container.fluid.fill-height > div > div > div > div > div > div.card__actions > button > div')
```

Comment:
Instead of this deep list of accessing css selectors we can add ids onto css components and then call those, it'll  look a lot cleaner.